pkg/api/api.go

```diff
@@ -151,7 +151,28 @@ type CheckCodeStatusResponse struct {
 	// UTC seconds since epoch.
 	LongExpiresAtTimestamp int64 `json:"longExpiresAtTimestamp,omitempty"`
 
-	Error     string `json:"error"`
+	Error     string `json:"error,omitempty"`
+	ErrorCode string `json:"errorCode,omitempty"`
+}
+
+// ExpireCodeRequest defines the parameters to request that a code be expired now.
+// This is called by the Web frontend.
+type ExpireCodeRequest struct {
```

Comment:
we should also sever this on the admin api as well

Reply:
that will be the next thing

Reply:
jk it's a now thing. Added that endpoint and a db unit test